### PR TITLE
[PXP-2318] chore(app-debug): Set log level based on config['DEBUG']

### PR DESCRIPTION
**Pull request by vpsx**
todo: update cdislogging tag and authutils tag

tldr: Updated cdislogging. In Fence, **stop using app.logger; only use `get_logger()` from cdislogging**, both to instantiate and to update logger objects. Supply the optional `log_level` argument to `get_logger()` to set logging levels, bearing in mind the tree structure of the loggers. You probably just want to set log level once in `app_config()`, and in all other modules, simply inherit the level and handlers from this parent logger by calling `get_logger()` with no `log_level` arg (which will set level to default NOTSET).

- [PXP-2318] is to set `app.debug` in `app_config()` based on `config["DEBUG"]`.
- However, toggling `app.debug` turns out to toggle not just DEBUG level logging but also INFO and WARNING levels.
- This is because flask 0.12 does this: https://github.com/pallets/flask/blob/1c2a2cd0014693f1e1f894e498c2dc50dbb44f66/flask/logging.py#L65-L89

- In both dev and prod, we would like to log at either DEBUG or INFO level depending on `config["DEBUG"]`.
- ~So if we want this behavior we need to always have `app.debug==True` and always have `app.config['LOGGER_HANDLER_POLICY']=='always'`, and then in `app_config()`, set the `logger.level`--not `app.debug`--based on `config["DEBUG"]`. So probably the story needs to be modified.~

- So, we are not going to use the app logger at all, and instead use only cdislogging `get_logger()`. 
- `get_logger()` should be used both to instantiate and to update logger objects. Supply `log_level` argument to `get_logger()` to set logging levels, bearing in mind the tree structure of the loggers.

- `app.debug` will still indeed be set based on `config["DEBUG"]`. Not a problem if we avoid using the flask app logger.

### Improvements
Set log level based on config['DEBUG']

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
The style in this PR agrees with [`black`](https://github.com/ambv/black). :heavy_check_mark:

This formatting comment was generated automatically by a script in [uc-cdis/wool](https://github.com/uc-cdis/wool).

File: fence/__init__.py [hunk 1]
```diff
@@ -173,6 +173,19 @@ def app_config(
         config["STORAGE_CREDENTIALS"], logger=app.logger
     )
 
+    # app.debug should always be True bc we want at least INFO lvl logging in dev+prod
+    # and flask 0.12 will set lvl to ERROR if app.debug is False
+    app.debug = True
+    logger.level = 10 if config["DEBUG"] == True else 20
+
+    if (
```
Comment (vpsx) [inline fence/__init__.py]: Ehh, probably unnecessary?

File: fence/__init__.py [hunk 2]
```diff
@@ -37,7 +37,7 @@
 import fence.blueprints.link
 import fence.blueprints.google
 
-from cdislogging import get_logger
+from cdislogging import get_logger, get_stream_handler
```
Comment (Avantol13) [inline fence/__init__.py]: I don't think we use `get_stream_handler`, can remove?
